libs/ballot-encoder/tsconfig.json

```diff
@@ -4,6 +4,7 @@
     "allowSyntheticDefaultImports": true,
     "esModuleInterop": true,
     "lib": ["dom", "dom.iterable", "esnext"],
+    "skipLibCheck": true,
```

Comment:
This works around this issue: https://stackoverflow.com/questions/75758154/how-to-fix-conflicting-abortsignal-in-node-modules-types-node-globals-d-ts-and. I didn't want to update `@types/node` because we're still using an old version of NodeJS and didn't want TS to promise APIs that aren't available.